test/test_loader.py

```diff
@@ -0,0 +1,20 @@
+import xypath.loader
```

Comment:
None of these tests run for me with `nosetests`. 

Think you should wrap them in a test class so they do!